packages/dso-toolkit/reference/render/conclusion--warning.html

```diff
@@ -1,10 +1,8 @@
 <ul class="dso-conclusion">
   <li class="dso-warning">
-    <strong>Neem contact op</strong> met uw gemeente voor omgevingsplanactiviteit op- en uitritten
-  </li>
+    <strong>Neem contact op</strong> met uw gemeente voor omgevingsplanactiviteit op- en uitritten</li>
```

Comment:
Hoort deze change bij het label?

Reply:
Dit gebeurt bij het genereren van de render files. Wellicht door de overstap van npm naar yarn en dan net een anders smaakje van een package?

Reply:
@tfrijsewijk komt jou dit bekend voor? Naar mijn idee hoort dit eigenlijk niet in het label PR

Reply:
Geen idee hoe dit komt.. Heb je wellicht `npm install` en/of `npm run start` gedaan? Want we doen alles met Yarn. Ik kan mij voorstellen dat er een versie mismatch met node_modules is waardoor dit gedrag komt..

Reply:
Ik denk dat het een merge error is tijdens upstreammergen `master` -> `next`

Reply:
Ik heb net een `yarn gulp reference:dom` op master gedaan, en zie ze daar ook

Reply:
Fixes (op 1 na) staan in : https://github.com/dso-toolkit/dso-toolkit/pull/878